README.md

```
@@ -69,6 +69,40 @@ swift run ExportOtherReminders /path/to/vault --cleanup
 swift run ExportOtherReminders --help
 ```
 
+### 4. ReSyncReminders
+
+**Clean vault and prepare for fresh sync**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)